modules/manage/pages/recovery-mode.adoc

```diff
@@ -0,0 +1,201 @@
+= Recovery Mode
```

Comment:
Not sure where this is going in the overall doc tree, but I assume under 'manage'